curve25519-dalek/Cargo.toml

```
@@ -65,6 +66,7 @@ default = ["alloc", "precomputed-tables", "zeroize"]
 alloc = ["zeroize?/alloc"]
 precomputed-tables = []
 legacy_compatibility = []
+group = ["group_crate", "rand_core"]
```

No need to rename the crate:

```suggestion
group = ["dep:group", "rand_core"]
```